docs/playbook/ingesting-data/005-ingesting-dynamo-db-tables.md

```diff
@@ -0,0 +1,178 @@
+They will need to create two roles, for both our pre-production and production AWS accounts.
+The roles will both need to have a trust policy that looks like this. One will hold the pre-production account number and one the production account number.
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "sts:AssumeRole",
+            "Principal": {
+                "AWS": "<account number of data platform account>"
+            }
+        }
+    ]
+}
+```
+Both roles should have the following policy attached to the role.
+Where ` -- TABLE ARNs HERE -- ` can be replaces by a list of ARNs for the tables that you want to ingest. 
+```
+{
+  "Version": "2012-10-17"
+  "Statement" [
+    {
+      "Effect" = "Allow"
+      "Action" = [
+        "dynamodb:List*",
+        "dynamodb:Get*",
+        "dynamodb:Query",
+        "dynamodb:Scan",
+        "dynamodb:BatchGet*",
+        "dynamodb:DescribeTable*",
+        "dynamodb:DescribeStream*",
+      ]
+      "Resource" = [
+        -- TABLE ARNs HERE --
+      ]
+    }
+  ]
+}
+```
+
+You will need the role ARN's of both roles, once they have been created, for the next step.
+
+## Saving the role ARN in the data platform accounts
+
+Complete this step in both pre-production & production AWS accounts.
+
+1. Login to AWS via [Hackneys SSO][hackney_sso].
+1. Navigate to [SSM parameter store][aws_console_ssm] in the AWS console.
+1. Click "Create parameter".
+1. Set the name of the parmeter to be something like `/where-the-data-is-coming-from/prod/role-arn-to-access-dynamodb-tables`.
+1. Keep note of the parameter name, as you will need it in the next section, and click "Create parameter".
+
+## Setting up the AWS Glue job
+
+1. Open the [Data Platform Project](https://github.com/LBHackney-IT/data-platform). If you don't have the correct permissions, you'll get a '404' error (see [prerequisites](#prerequisites)).
+2. Navigate to the main `terraform` directory and add a new file named after the dataset you are ingesting, for example `30-mtfh-tables-ingestion.tf`. Click on "Add file" then "Create new file" to do this.
+3. Load the role ARN saved in the [previous section](#saving-the-role-arn-in-the-data-platform-accounts) in to terraform.
+  - Copy this example code block into the file.
+  - Rename `role_arn_to_access_housing_tables` to be relevent to the kind of tables you are ingesting.
+  - Change the name `/where-the-data-is-coming-from/prod/role-arn-to-access-dynamodb-table` to be the name of the parameter you created in the previous section.
+
+  ```tf
+  data "aws_ssm_parameter" "role_arn_to_access_housing_tables" {
+    name = "/where-the-data-is-coming-from/prod/role-arn-to-access-dynamodb-table"
+  }
+  ```
+
+4. Create the glue job & crawler.
+Copy the example terraform module below into the file.
+This is example terraform to create a glue job in the Data Platform account.
```

Comment:
Could the example terraform be moved to near the top of this section? It could make the reading flow better, rather than having to scroll down, then scroll back up to learn what you need to change